code-scan.js

```diff
@@ -170,9 +173,11 @@ async function run() {
   }
 
   if (issueBody.length > 0) {
-    console.log(`Creating a new comment in issue [${issue_number}] to indicate [${issueBody}]`)
-    // create an comment in the issue to indicate why a manual check is needed
-    octokitRequest("createIssueComment", {
+    console.log(`Creating a new comment in issue [${issue_number}] in repo [${owner}/${repo}] to indicate status: [${issueBody}]`)
+    // create an comment in the issue to indicate why a manual check is needed. uses different client!
+    issue_octokit.rest.issues.createComment("createIssueComment", {
+      owner,
```

Comment:
Owner and repo in this context are probably not correct since they're not of this repo. You can just pass them here:
      owner: 'asml-actions',
      repo: 'github-fork-updater',